eidas-saml-parser/src/main/java/uk/gov/ida/notification/eidassaml/ResponseDto.java

```diff
@@ -10,10 +10,26 @@
     @JsonProperty
     public String issuer;
 
+    @JsonProperty
+    public String connectorPublicEncryptionKey;
```

Comment:
Could we send back an X509Certificate instead of a String for the cert, as per the shared DTO `EidasSamlParserResponse` i.e.

`@NotNull
    @JsonProperty
    @JsonSerialize(using = X509CertificateSerializer.class)
    @JsonDeserialize(using = X509CertificateDeserializer.class)
    private X509Certificate connectorEncryptionPublicCertificate;`